src/generators/java/renderers/ClassRenderer.ts

```diff
@@ -82,14 +82,23 @@ export const JAVA_DEFAULT_CLASS_PRESET: ClassPresetType<JavaOptions> = {
   self({ renderer }) {
     return renderer.defaultSelf();
   },
-  property({ property }) {
+  property({ renderer, property }) {
+    const constValue = renderer.getConstValue(property);
+
+    if (constValue) {
+      return `private final ${property.property.type} ${property.propertyName} = ${property.property.type}.${constValue.key};`;
+    }
```

Comment:
This assumes that constants are only from an enum, which definitely is not the case in the future as all models can have default values (I think).

Reply:
Maybe we should move this as part of the renderer so we can just call `${renderer.getConstantValue(property)}` instead of `${property.property.type}.${constValue.key}`.